targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_KW41Z/TARGET_RAPIDIOT/PeripheralPins.c

```diff
@@ -0,0 +1,136 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2013 ARM Limited
```

Comment:
can you add SPDX specifier to the new files please?

`SPDX-License-Identifier: Apache-2.0` (files in platform for instance contain it already)

Reply:
Also fix the year (new files should have 2018 in this case)

Reply:
Thanks, copyright on the new files updates to the right year and added the Apache identifier